### ISSUE TITLE
add a news item

### ISSUE THREAD

**Issue by ValWood** [japonicusdb/japonicus-config#26]
Introducing JaponicusDB beta  May 26th(or something)

Welcome to the beta version of JaponicusDB, the model organism database for _S. japonicus_ created using the PomBase software. If you are unfamiliar with PomBase this *linky Methods Book chapter* provides a guide to PomBase data organization and features, and this *linky "quickstart guide" https://www.pombase.org/documentation/JBrowse_quick_start describes JBrowse basics.  Until the JaponicusDB FAQ is established, many items in the PomBase FAQ will also be applicable to JaponicusDB https://www.pombase.org/faq/all-frequently-asked-questions

**Comment by ValWood** [comment]
@mah11  could you edit. May as well use the news item spot to help the newbies.

**Comment by kimrutherford** [comment]
Thanks for the news item.  I don't have a place to put them yet.  I'll work on it.

See also: #3

**Comment by kimrutherford** [comment]
> Thanks for the news item. I don't have a place to put them yet.

News for japonicus now goes in `website/src/docs/news.JaponicusDB/`

> I'm uploading a file here,

Thanks.  I've added `welcome-to-japonicusdb.md` to the `news.JaponicusDB/` directory and given it today's date.  I set the front page flag.

That news item is now on the front page and in the archive:
  http://japonicusdb.kmr.nz/
  http://japonicusdb.kmr.nz/news

For now at least, I'll need to manually kick off a new build if a news item is added or changes.
